Cargo.lock

```diff
@@ -1156,6 +1181,12 @@ dependencies = [
  "futures 0.1.29",
 ]
 
+[[package]]
+name = "futures-timer"
```

Comment:
What crate needs this, I've seen some recent bugs with this crate. As in UB.

Reply:
I just pinged the author of kube, lets see what they say but this shouldn't block this.